_includes/pbc/all/install-features/202304.0/install-glue-api/install-backend-api-glue-json-api-convention.md

```diff
@@ -0,0 +1,141 @@
+
+class DocumentationGeneratorApiDependencyProvider extends SprykerDocumentationGeneratorApiDependencyProvider
+{
+    /**
+     * @var string
+     */
+    protected const GLUE_BACKEND_API_APPLICATION_NAME = 'backend';
+
+    /**
+     * @param \Spryker\Glue\DocumentationGeneratorApi\Expander\ContextExpanderCollectionInterface $contextExpanderCollection
+     *
+     * @return \Spryker\Glue\DocumentationGeneratorApi\Expander\ContextExpanderCollectionInterface
+     */
+    protected function getContextExpanderPlugins(ContextExpanderCollectionInterface $contextExpanderCollection): ContextExpanderCollectionInterface
+    {
+        $apiApplications = [];
+        foreach ($this->getApiApplicationProviderPlugins() as $apiApplicationProviderPlugin) {
+            $apiApplications[] = $apiApplicationProviderPlugin->getName();
+        }
+        $contextExpanderCollection->addApplications($apiApplications);
+
+        $contextExpanderCollection->addExpander(new BackendRelationshipPluginsContextExpanderPlugin(), [static::GLUE_BACKEND_API_APPLICATION_NAME]);
+
+        return $contextExpanderCollection;
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+In order to make sure that `RelationshipPluginsContextExpanderPlugin` is setup correctly,
+attempt to generate the documentation for `backend`. Do so by passing the optional `--application` parameter:
+```
+vendor/bin/glue api:generate:documentation --application backend
+```
+Make sure only the "backend" application documentation was generated with related relationships.
+
+{% endinfo_block %}
+
+**src/Pyz/Glue/GlueJsonApiConvention/GlueJsonApiConventionDependencyProvider.php**
+
+```
+<?php
+
+namespace Pyz\Glue\GlueJsonApiConvention;
+
+use Spryker\Glue\GlueJsonApiConvention\GlueJsonApiConventionDependencyProvider as SprykerGlueJsonApiConventionDependencyProvider;
+use Spryker\Glue\GlueBackendApiApplicationGlueJsonApiConventionConnector\Plugin\GlueJsonApiConvention\BackendApiRelationshipProviderPlugin;
+
+class GlueJsonApiConventionDependencyProvider extends SprykerGlueJsonApiConventionDependencyProvider
+{
+    /**
+     * @return array<\Spryker\Glue\GlueJsonApiConventionExtension\Dependency\Plugin\RelationshipProviderPluginInterface>
+     */
+    public function getRelationshipProviderPlugins(): array
+    {
+        return [
+            new BackendApiRelationshipProviderPlugin(),
+        ];
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+To verify that everything is set up correctly, and you can access the endpoint, see [How to create a backend resource](/docs/scos/dev/glue-api-guides/{{site.version}}/decoupled-glue-infrastructure/how-to-guides/routing/how-to-create-a-backend-resource.html) or [How to create a backend resource](/docs/scos/dev/glue-api-guides/{{site.version}}/decoupled-glue-infrastructure/how-to-guides/routing/how-to-create-a-backend-resource.html).
```

Comment:
These links are identical. Remove one of them or replace one of them with the proper one.